### PR TITLE
Final adjustments and corrections for schema validation

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1340

This PR:

- Adds missing id for "DDC-Sachgruppen der ZDB"
- Adds a variation in the skosConcept schema to either reqiure id AND label or jus label or notation without id.
- Fix a formal error in the json schema, instead of tuple validation, the items element has to be an object instead of an array, otherwise it would only check the first element.
- Adds a missing value to the type schema.

### CODE REVIEW

File: src/main/resources/alma/fix/subjects.fix [hunk 1]
```diff
@@ -296,8 +296,10 @@ do list(path:"084??", "var":"$i")
             if any_match("$j","\\d{3}(\\.\\d{1,3})?|[BKS]")
               if any_equal("$i.q","DE-600")
                 add_field("subject[].$last.source.label","DDC-Sachgruppen der ZDB")
+                add_field("subject[].$last.source.id","https://zeitschriftendatenbank.de/fileadmin/user_upload/ZDB/pdf/zdbformat/5080.pdf")
```
Comment (acka47) [inline src/main/resources/alma/fix/subjects.fix:299]: Interesting choice of `id` for this source. While I like the pragmatism, this sets a new precedent linking to the description of a controlled vocabulary in a documentation PDF. As I have no better idea, I guess we can leave it for now.

Comment (acka47) [reply src/main/resources/alma/fix/subjects.fix:299]: On the other hand, we should probably better leave out the `id` field and add the documentation link in another field, e.g. `seeAlso`.

**Review by acka47 — CHANGES_REQUESTED** [review]
The change to `skosConcept.json` looks good but we have to discuss the `id` for DDC-Sachgruppen der ZDB.

File: src/main/resources/alma/fix/subjects.fix (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/fix/subjects.fix:299]: I already had a bad feeling a while ago about id: https://github.com/hbz/lobid-resources/issues/1848#issuecomment-1707846009

Comment (TobiasNx) [reply src/main/resources/alma/fix/subjects.fix:299]: But we decided off board to go on for a lack of a better id.

Comment (acka47) [reply src/main/resources/alma/fix/subjects.fix:299]: Ok, then let's go. +1

**Review by acka47 — APPROVED** [review]

**Pull request merged**